lib/vyasa/medium/event.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Vyasa.Medium.Event do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Vyasa.Written.{Verse, Source}
+  alias Vyasa.Medium.Voice
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  schema "events" do
+    field :origin, :integer
+    field :duration, :integer
+    field :phase, :string
+
+    embeds_many :fragments, EventFrag do
```

Comment:
NB: fragments will be untested for the next few bits, just something to keep at the back of our minds, in case there's a better way to handle fragments